client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -126,10 +126,19 @@ const DataFilesToolbar = ({ scheme, api }) => {
   };
 
   const download = () => {
-    dispatch({
-      type: 'DATA_FILES_DOWNLOAD',
-      payload: { file: selectedFiles[0] }
-    });
+    canDownload &&
+      dispatch({
+        type: 'DATA_FILES_DOWNLOAD',
+        payload: { file: selectedFiles[0] }
+      });
+    !canDownload &&
+      dispatch({
+        type: 'DATA_FILES_TOGGLE_MODAL',
+        payload: {
+          operation: 'downloadMessage',
+          props: {}
+        }
+      });
```

Comment:
__Suggestion__
Change this conditional syntax to use `if` and `else`.

<details><summary>Why?</summary>

- The `falseyValue &&` syntax is less intuitive than `if` and `else`.
- The `falseyValue &&` syntax seems—in this codebase—reserved for JSX.
    - JSX makes conditional markup otherwise cumbersome.

</details>
<details><summary>How?</summary>

One way is:

```suggestion
    if (canDownload) {
      dispatch({
        type: 'DATA_FILES_DOWNLOAD',
        payload: { file: selectedFiles[0] }
      });
    } else {
      dispatch({
        type: 'DATA_FILES_TOGGLE_MODAL',
        payload: {
          operation: 'downloadMessage',
          props: {}
        }
      });
    }
```

</details>